package com.quize.quiz_service.entity;


import lombok.Data;

@Data
public class QuestionWrapper {

    private Integer id;

    public QuestionWrapper(Integer id, String category, String questionTitle, String option2, String option1, String option3, String option4) {
        this.id = id;
        this.category = category;
        this.questionTitle = questionTitle;
        this.option2 = option2;
        this.option1 = option1;
        this.option3 = option3;
        this.option4 = option4;
    }

    private String category;
    private String questionTitle;
    private String option1;
    private String option2;
    private String option3;
    private String option4;
}
